### PR TITLE
Fix PDF invoice generator footer which only shows a %s

### LINKED ISSUE #32485: PDF invoice generator footer is not showing the correct information only show a %s
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

In the PDF invoice generator footer is not showing the correct information only shows a %s instead of the phone
image

`        $shop_address = $this->getShopAddress();

        $id_shop = (int) $this->shop->id;

        $this->smarty->assign([
            'available_in_your_account' => $this->available_in_your_account,
            'shop_address' => $shop_address,
            'shop_fax' => Configuration::get('PS_SHOP_FAX', null, null, $id_shop),
            'shop_phone' => Configuration::get('PS_SHOP_PHONE', null, null, $id_shop),
            'shop_email' => Configuration::get('PS_SHOP_EMAIL', null, null, $id_shop),
            'free_text' => Configuration::get('PS_INVOICE_FREE_TEXT', (int) Context::getContext()->language->id, null, $id_shop),
        ]);
`

### Expected behavior

It should add the phone number instead of  %s

### Steps to reproduce

Go to orders > Invoice> generate a PDF

### PrestaShop version(s) where the bug happened

8.0.4

### PHP version(s) where the bug happened

7.4

### If your bug is related to a module, specify its name and its version

PDF Generator

### Your company or customer's name goes here (if applicable).

_No response_

### PR DESCRIPTION
**Pull request by tesshsu**
…ion only show a %s

| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Bug fix once generate pdf footer shop phone and shop fax information not generated by sprinf, the twig  l function may not work as expected.
| Type?             | bug fix
| Category?         | FO
| BC breaks?        |  no
| Deprecations?     | no
| How to test?      | Go to BO->invoice->generate pdf, then check invoice pdf footer has show correctly shop phone and fax, you have to go to BO add your shop phone and fax number to test
| Fixed ticket?     | Fixes #32485

### CODE REVIEW

**Comment by tesshsu** [comment]
After fixing the information shop phone and shop fax should be shown correctly:

**Comment by kpodemski** [comment]
Hi @tesshsu 

I believe it is a workaround, not a real solution, see, before, you could change the wording, for instance:
"The phone number is %s", right now, it always has to be: "Phone: phone number". We need to investigate why `sprintf` does not work in the PDF.

**Comment by tesshsu** [comment]
@kpodemski Thanks for the feedback, I did not see global scoop, hence I found the TranslateCore Class and enhance the condition to translate sprinf array part, let me know if need further enhancement, tks

**Review by kpodemski — DISMISSED** [review]

**Review by kpodemski — APPROVED** [review]

File: pdf/footer.tpl [hunk 1]
```diff
@@ -33,13 +33,13 @@
...
+        {if !empty($shop_phone)}
+          {l s='Tel: %s' sprintf=[$shop_phone|escape:'html':'UTF-8'] d='Shop.Pdf' pdf='true'}
+        {/if}
 
-				{if !empty($shop_fax)}
-					{l s='Fax: %s' sprintf=[$shop_fax|escape:'html':'UTF-8'] d='Shop.Pdf' pdf='true'}
-				{/if}
+        {if !empty($shop_fax)}
+          {l s='Fax: %s' sprintf=[$shop_fax|escape:'html':'UTF-8'] d='Shop.Pdf' pdf='true'}
+        {/if}
```
Comment (SharakPL) [inline pdf/footer.tpl:42]: Why this change? Indentations look incorrect now

Comment (matks) [reply pdf/footer.tpl:42]: Usually when I see such diff in GitHub, it's white characters replaced by another white characters. Like spaces and tabs.

Here I think tabs were replaced by spaces

**Review by matks — APPROVED** [review]

**Review by AchrafKchaou1991 — APPROVED** [review]
Hello@
I tested this PR :
The issue is well fixed :heavy_check_mark: 

[PR32500.webm](https://github.com/PrestaShop/PrestaShop/assets/131356155/9b0737e7-733f-4d07-bb72-3bf205c3a412)




The automated tests are OK :heavy_check_mark: https://github.com/AchrafKchaou1991/ga.tests.ui.pr/actions/runs/5119554440
PR :heavy_check_mark: 
Thank you
